### ISSUE TITLE
HMIP Wassermelder

### ISSUE THREAD

**Issue by backons** [iobroker-community-adapters/ioBroker.hmip#45]
Folgendes bekomme ich im LOG nach dem Start:
unkown channel type - DEVICE_INCORRECT_POSITIONED - {"id":"3014****","homeId":"782****","label":"Wassersensor Keller","lastStatusUpdate":155***,"typ...

**Comment by jogibear9988** [comment]
ja das ist nicht schlimm, den kanaltyp habe ich noch nicht realisiert

**Comment by saepfle** [comment]
Hi, is there a very very rough time estimation when the channel for the water sensor HMIP-SWD could be available?

Thank you :-)

**Comment by steckenpferd** [comment]
Any news?

**Comment by jogibear9988** [comment]
json dazu?

**Comment by steckenpferd** [comment]
@backons könntest du deine LOG Daten für den HMIP-SWD zur Verfügung stellen - dann könnte man den Channel hinzufügen

**Comment by backons** [comment]
@steckenpferd  ja das könnte ich ... Was genau wird benötigt und wo soll ich das hinschicken

**Comment by steckenpferd** [comment]
Auf der Startseite steht der Hinweis:

Switch adapter logging in ioBroker to silly mode and add the json of the device which is printed to the log in the issue. I may also need a json of a state change.

s. z.B. hier: https://github.com/iobroker-community-adapters/ioBroker.hmip/issues/50
und hier:

https://github.com/iobroker-community-adapters/ioBroker.hmip/issues/53

**Comment by backons** [comment]
hmip.0 unkown channel type - DEVICE_INCORRECT_POSITIONED - {"id":"3014F711A0001898A*****","homeId":"782aac6e-1edd-4116-b178-*************","label":"Wassersensor Keller","lastStatusUpdate":1556971132667,"type":"WATER_SENSOR","functionalChannels":{"0":{"label":"","deviceId":"3014F711A0001898A*******","index":0,"groupIndex":0,"functionalChannelType":"DEVICE_INCORRECT_POSITIONED","groups":["1bbfa801-8574-4383-93bc-94b8f0afe3dc"],"unreach":false,"lowBat":false,"routerModuleEnabled":false,"routerModuleSupported":false,"rssiDeviceValue":-58,"rssiPeerValue":null,"configPending":false,"dutyCycle":false,"incorrectPositioned":false},"1":{"label":"","deviceId":"3014F711A0001898A*******","index":1,"groupIndex":1,"functionalChannelType":"WATER_SENSOR_CHANNEL","groups":["1ce10314-8de7-452d-a999-c9efb47ad799"],"waterlevelDetected":false,"moistureDetected":false,"acousticWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","sirenWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","inAppWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","acousticAlarmSignal":"FREQUENCY_RISING","acousticAlarmTiming":"PERMANENT"}},"oem":"eQ-3","manufacturerCode":1,"firmwareVersion":"1.0.2","updateState":"UP_TO_DATE","firmwareVersionInteger":65538,"liveUpdateState":"LIVE_UPDATE_NOT_SUPPORTED","availableFirmwareVersion":"0.0.0","serializedGlobalTradeItemNumber":"3014F711A0001898A9926A1D","modelType":"HmIP-SWD","modelId":353,"permanentlyReachable":false}

**Comment by Apollon77** [comment]
Bei mir auch aufgetreten:

```
2019-09-21 22:01:44.894  - info: hmip.0 unkown channel type - DEVICE_INCORRECT_POSITIONED - {"id":"3014F711A0001898A992629D","homeId":"b3d12282-1464-43dd-8d8c-20ff4b6bad08","label":"Wassersensor","lastStatusUpdate":1569095947062,"type":"WATER_SENSOR","functionalChannels":{"0":{"label":"","deviceId":"3014F711A0001898A992629D","index":0,"groupIndex":0,"functionalChannelType":"DEVICE_INCORRECT_POSITIONED","groups":["e2753fd9-8d1a-4848-a6db-51d0e66eda54"],"unreach":false,"lowBat":false,"routerModuleEnabled":false,"routerModuleSupported":false,"rssiDeviceValue":-70,"rssiPeerValue":null,"configPending":false,"dutyCycle":false,"deviceOverloaded":false,"coProUpdateFailure":false,"coProFaulty":false,"coProRestartNeeded":false,"deviceUndervoltage":false,"deviceOverheated":false,"temperatureOutOfRange":false,"supportedOptionalFeatures":{"IFeatureDeviceCoProRestart":false,"IFeatureDeviceOverheated":false,"IFeatureDeviceUndervoltage":false,"IFeatureDeviceCoProUpdate":false,"IFeatureDeviceCoProError":false,"IFeatureDeviceOverloaded":false,"IFeatureDeviceTemperatureOutOfRange":false},"incorrectPositioned":false},"1":{"label":"","deviceId":"3014F711A0001898A992629D","index":1,"groupIndex":1,"functionalChannelType":"WATER_SENSOR_CHANNEL","groups":["287bbac6-cced-427a-b57d-535a5dabae13","8769e977-9fed-435c-add4-3dd701abedd6"],"waterlevelDetected":false,"moistureDetected":false,"acousticWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","sirenWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","inAppWaterAlarmTrigger":"WATER_MOISTURE_DETECTION","acousticAlarmSignal":"FREQUENCY_RISING","acousticAlarmTiming":"PERMANENT"}},"oem":"eQ-3","manufacturerCode":1,"updateState":"UP_TO_DATE","firmwareVersion":"1.0.2","firmwareVersionInteger":65538,"liveUpdateState":"LIVE_UPDATE_NOT_SUPPORTED","availableFirmwareVersion":"1.0.2","modelType":"HmIP-SWD","modelId":353,"serializedGlobalTradeItemNumber":"3014F711A0001898A992629D","permanentlyReachable":false}
```

**Comment by jogibear9988** [comment]
Fixed with #75
